README.md

```diff
@@ -1,54 +1,69 @@
-  <a href="https://developer.1password.com/docs/cli/shell-plugins/contribute"><b>Contribution docs</b></a>
+  <a href="https://developer.1password.com/docs/cli/shell-plugins/"><img alt="Get Started" src="https://user-images.githubusercontent.com/45081667/226940040-16d3684b-60f4-4d95-adb2-5757a8f1bc15.png" height="37" /></a>
+  <a href="https://developer.1password.com/docs/cli/shell-plugins/contribute"><img alt="Build" src="https://user-images.githubusercontent.com/45081667/226941089-473be407-5417-48e3-8710-55b2d4ec761a.png" height="37" /></a>
 </p>
 <br/>
 
 ---
 
 No more plaintext credentials in your home directory. Automatically authenticate every CLI you use with [1Password CLI](https://developer.1password.com/docs/cli/) + [Shell Plugins](https://developer.1password.com/docs/cli/shell-plugins/). Approve temporary credential usage in your terminal with biometrics.
 
-## 🪄 Usage
-
- <picture>
-   <source srcset="https://developer.1password.com/videos/aws.webp" type="image/webp" />
-   <img src="https://user-images.githubusercontent.com/7430639/205337855-41604aca-0ddb-4eab-a5f0-fb9107e09d8d.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
+## 🪄 See it in action!
+<p align="center">
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/45081667/227191964-9629476d-a49e-475d-b8cb-2115c302025d.gif">
+   <img src="https://user-images.githubusercontent.com/45081667/227197994-6fdb2cad-c240-4cb7-ba4c-77f2483606ab.gif" alt="Example of 1Password Shell Plugins with AWS: user runs an `aws` command, a Touch ID prompt shows up, and `aws` is automatically authenticated" style="max-width: 100%; display: inline-block;" />
 </picture>
+</p>
+
+
+## 🚀 Get started with the available plugins
+
+<p align="center">
+<a href="https://developer.1password.com/docs/cli/shell-plugins/">
+<picture>
+
+  <source media="(prefers-color-scheme: dark)" srcset="https://user-images.githubusercontent.com/45081667/226968760-70d3f6b0-a3eb-4c75-a674-6fd136d7149a.png">
 
-## 🚀 Get started
+  <img alt="Picture changing depending on light mode." src="https://user-images.githubusercontent.com/45081667/226969008-0a3f7537-7942-442f-9170-18b008a6574c.png">
 
-* Get started with [1Password Shell Plugins](https://developer.1password.com/docs/cli/shell-plugins)
-* Get started with [creating your own plugins](https://developer.1password.com/docs/cli/shell-plugins/contribute) <sup><b>[BETA]</b></sup>
+</picture>
+See all...
+</a>
+</p>
 
-## 👫 Contributing
+<br/>
 
-Is your favorite CLI not listed yet? Learn how to [build a shell plugin](https://developer.1password.com/docs/cli/shell-plugins/contribute) yourself and [open a PR](https://github.com/1Password/shell-plugins/pulls)!
+## 🔩 Don’t see yours? Contribute! <sup><b><a href="#-beta-notice">[BETA]</a></b></sup>
```

Comment:
I noticed that the "Contributions Beta Notice" section was moved to the CONTRIBUTING.md file but the hyperlink isn't removed here. Should we remove the link completely, or link to [here](https://github.com/1Password/shell-plugins/blob/andi_t/readme-revamp/CONTRIBUTING.md#-contributions-beta-notice)? I am thinking the latter.

Reply:
Very well spotted @arunsathiya , thanks! Done!